### PR TITLE
Fluent fixes 4

### PR DESCRIPTION
**Pull request by kikostadinov**
Related to: https://github.com/telerik/kendo-themes-private/issues/26

Wizard
- `Paddings of .k-wizard-buttons should be like .k-wizard-content. Currently, .k-wizard-buttons haven't paddings and 'Steps' text doesn't seem properly aligned to the content.`

Stepper
- `The background color of the current item should be #005A9E by design`
- `Disabled state of 'default' step(not 'done' ) doesn't follow the design - there is not visual indication that the step is disabled`

Splitter
- `bg color of the focused item should be blue - currently it's grey`

ChunkProgressBar
- `The space between the chunks should be 1px, not 3 px.`

ImageEditor
- `The toolbar bottom padding is 8px. By design it is 5px.` - sync'ed with design team and should remain 8px
- `Toolbar column gap is 8px. By design it is 6 px.` - sync'ed with design team and should remain 8px

### CODE REVIEW

File: packages/fluent/scss/wizard/_layout.scss [hunk 1]
```diff
@@ -45,8 +45,6 @@
         }
 
         .k-wizard-content {
-            padding-inline: var( --kendo-wizard-content-padding-x, #{$kendo-wizard-content-padding-x} );
-            padding-block: var( --kendo-wizard-content-padding-y, #{$kendo-wizard-content-padding-x} );
```
Comment (Juveniel) [inline packages/fluent/scss/wizard/_layout.scss:49]: either set the variables to 0px and allow the option for customization or remove both the _layout styles and the variables as well.

File: packages/fluent/scss/stepper/_layout.scss [hunk 2]
```diff
@@ -131,7 +131,7 @@
         }
 
         .k-step-current .k-step-label {
-            font-weight: bold;
+            font-weight: 600;
```
Comment (Juveniel) [inline packages/fluent/scss/stepper/_layout.scss:134]: extract a variable that uses the --kendo-font-weight-bold var

File: packages/fluent/scss/stepper/_layout.scss [hunk 3]
```diff
@@ -145,14 +145,19 @@
...
 
             .k-step-label-optional {
                 color: inherit;
             }
+
+            &.k-step-current {
+                .k-step-label {
+                    font-weight: 400;
+                }
+            }
```
Comment (Juveniel) [inline packages/fluent/scss/stepper/_layout.scss:160]: probably this ?

```suggestion
.k-step-label {
   font-weight: normal;
}
```

**Review by Juveniel — CHANGES_REQUESTED** [review]

File: packages/fluent/scss/splitter/_variables.scss [hunk 4]
```diff
@@ -56,10 +56,10 @@ $kendo-splitbar-hover-text: $kendo-splitbar-text !default;
 
 /// Selected background color of the splitter splitbar.
 /// @group splitter
-$kendo-splitbar-selected-bg: var( --kendo-selected-bg, initial ) !default;
+$kendo-splitbar-selected-bg: get-theme-color-var( primary-100 ) !default;
 /// Selected text color of the splitter splitbar.
 /// @group splitter
-$kendo-splitbar-selected-text: var( --kendo-selected-text, initial ) !default;
+$kendo-splitbar-selected-text: $white !default;
```
Comment (PaskalevStoyan) [inline packages/fluent/scss/splitter/_variables.scss:62]: use `var( --kendo-component-bg, inherit )` here

**Review by PaskalevStoyan — DISMISSED** [review]

**Comment by Juveniel** [comment]
all commits should be fix(componentName): scenario in fluent theme, as the theme is already released

File: packages/fluent/scss/stepper/_layout.scss [hunk 5]
```diff
@@ -131,7 +131,7 @@
         }
 
         .k-step-current .k-step-label {
-            font-weight: bold;
+            font-weight: var( --kendo-stepper-current-label-font-weight, #{$kendo-stepper-current-label-font-weight} );;
```
Comment (Juveniel) [inline packages/fluent/scss/stepper/_layout.scss:134]: remove second semicolon

**Review by Juveniel — CHANGES_REQUESTED** [review]

**Review by Juveniel — APPROVED** [review]

**Pull request merged**